handlers-converters/handlers/field-mapping/src/test/java/com/github/cafdataprocessing/worker/policy/handlers/fieldmapping/FieldMappingHandlerTest.java

```diff
@@ -150,8 +151,53 @@ public void testFieldNameSwapWithCommonTargetName() throws IOException
         Assert.assertTrue(document.getMetadata().get("abc").isEmpty());
     }
 
+    @Test
+    public void testFieldNameMappingOfEncodedField() throws IOException
+    {
+        //Arrange
+        final Document document = setupDocument();
+        final Policy testPolicy = new Policy();
+        testPolicy.id = 1L;
+        testPolicy.details = new ObjectMapper().readTree("{"
+            + "  \"mappings\":"
+            + "  {"
+            + "    \"abc\": \"xyz\","
+            + "    \"def\": \"pqr\","
+            + "    \"pqr\": \"xyz\","
+            + "    \"jkl\": \"wrx\""
+            + "  }"
+            + "}");
+        final Long testColSeqId = 1L;
+        try (final ByteArrayInputStream stream = new ByteArrayInputStream("This is a test string".getBytes(StandardCharsets.UTF_8))) {
+            document.getStreams().put("abc", stream);
```

Comment:
The try with isn't needed here. Close on a ByteArrayInput stream doesn't do anything.

https://docs.oracle.com/javase/7/docs/api/java/io/ByteArrayInputStream.html